cli.c

```diff
@@ -82,22 +82,18 @@ struct cli_options *cli_options_parse(int argc, char *argv[]) {
   }
 
   const struct option longopts[] = {
-      {"socket-group", required_argument, NULL, CLI_OPT_SOCKET_GROUP},
-      {"vmnet-mode", required_argument, NULL, CLI_OPT_VMNET_MODE},
-      {"vmnet-interface", required_argument, NULL,
-       CLI_OPT_VMNET_INTERFACE},
-      {"vmnet-gateway", required_argument, NULL, CLI_OPT_VMNET_GATEWAY},
-      {"vmnet-dhcp-end", required_argument, NULL,
-       CLI_OPT_VMNET_DHCP_END},
-      {"vmnet-mask", required_argument, NULL, CLI_OPT_VMNET_MASK},
-      {"vmnet-interface-id", required_argument, NULL,
-       CLI_OPT_VMNET_INTERFACE_ID},
-      {"vmnet-nat66-prefix", required_argument, NULL,
-       CLI_OPT_VMNET_NAT66_PREFIX},
-      {"pidfile", required_argument, NULL, 'p'},
-      {"help", no_argument, NULL, 'h'},
-      {"version", no_argument, NULL, 'v'},
-      {0, 0, 0, 0},
+    {"socket-group",        required_argument,  NULL,   CLI_OPT_SOCKET_GROUP},
+    {"vmnet-mode",          required_argument,  NULL,   CLI_OPT_VMNET_MODE},
+    {"vmnet-interface",     required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE},
+    {"vmnet-gateway",       required_argument,  NULL,   CLI_OPT_VMNET_GATEWAY},
+    {"vmnet-dhcp-end",      required_argument,  NULL,   CLI_OPT_VMNET_DHCP_END},
+    {"vmnet-mask",          required_argument,  NULL,   CLI_OPT_VMNET_MASK},
+    {"vmnet-interface-id",  required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE_ID},
+    {"vmnet-nat66-prefix",  required_argument,  NULL,   CLI_OPT_VMNET_NAT66_PREFIX},
+    {"pidfile",             required_argument,  NULL,   'p'},
+    {"help",                no_argument,        NULL,   'h'},
+    {"version",             no_argument,        NULL,   'v'},
+    {0, 0, 0, 0},
```

Comment:
We should use `clang-format` or something for maintaining the format.

Does `clang-format` or something support keeping this whitespacing?

Reply:
No idea but even if not having automatic formatting is better.

Reply:
All the default styles (LLVM, GNU, Google, Chromium, Microsoft, Mozilla, WebKit) destroy the nice table. Maybe there is an option to align tables, or to disable formatting of part of a file.

Reply:
I haven't tested it, but according to [Disabling Formatting on a Piece of Code](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code) you can disable it for a block like this:

```c
// clang-format off
  const struct option longopts[] = {
    {"socket-group",        required_argument,  NULL,   CLI_OPT_SOCKET_GROUP},
    {"vmnet-mode",          required_argument,  NULL,   CLI_OPT_VMNET_MODE},
    {"vmnet-interface",     required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE},
    {"vmnet-gateway",       required_argument,  NULL,   CLI_OPT_VMNET_GATEWAY},
    {"vmnet-dhcp-end",      required_argument,  NULL,   CLI_OPT_VMNET_DHCP_END},
    {"vmnet-mask",          required_argument,  NULL,   CLI_OPT_VMNET_MASK},
    {"vmnet-interface-id",  required_argument,  NULL,   CLI_OPT_VMNET_INTERFACE_ID},
    {"vmnet-nat66-prefix",  required_argument,  NULL,   CLI_OPT_VMNET_NAT66_PREFIX},
    {"pidfile",             required_argument,  NULL,   'p'},
    {"help",                no_argument,        NULL,   'h'},
    {"version",             no_argument,        NULL,   'v'},
    {0, 0, 0, 0},
  };
// clang-format on
```

Reply:
I opened #104